code/__DEFINES/obj_flags.dm

```diff
@@ -28,6 +28,7 @@
 #define IMMUTABLE_SLOW			(1<<10) // When players should not be able to change the slowdown of the item (Speed potions, etc)
 #define IN_STORAGE				(1<<11) //is this item in the storage item, such as backpack? used for tooltips
 #define SURGICAL_TOOL			(1<<12)	//Tool commonly used for surgery: won't attack targets in an active surgical operation on help intent (in case of mistakes)
+#define NO_CHRISTMAS			(1<<13)	//When something should not be attainable from a christmas present
```

Comment:
proposal: Instead of doing this, give each of the skill capes funny default names/descriptions to indicate that they're fake. Make earning a skill cape legitimately modify its name/description to the "correct" ones.

That way, if someone spawns in a skill cape (or opens one from a present) without "earning" it, everyone can laugh at them, and the christmas present's schtick of "could contain ANY item in the game (that isn't abstract or spriteless or something)!" doesn't get harmed.

Reply:
sounds good

Reply:
I'll do ya one better, i'll just make it so you can't equip it if your skill level is too low.
Fuck it, i want people to see someone wearing this and know without a doubt that they're a beast of a gamer